debian

```diff
@@ -12,8 +12,8 @@
 
 NAME="example"
 PATH="/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin"
-APPDIR="/usr/bin"
-APPBIN="python"
+APPDIR="/"
```

Comment:
The / directory is the default directory from where any daemon runs. You can choose to change to another directory if your daemon works with relative paths.